makefile

```diff
@@ -223,22 +223,22 @@ notebook: requirements dev-requirements
 # Making sure the examples are running, without validating their outputs.
 	@$(JUPYTER) nbextension enable --py widgetsnbextension
 	@echo "+++ Number of notebooks to execute: $$(find ./docs/source/examples -name "*.ipynb" | wc -l) +++"
-	@$(JUPYTER) nbconvert \
-		--execute $$(find ./docs/source/examples -name "*.ipynb") \
-		--to notebook \
-		--stdout > /dev/null
-# For now, because of plotly - disabling the nbval and just validate that the notebooks are running
+	@echo "+++ Executing notebooks in $(PWD) +++"
+	@find ./docs/source/examples -name "*.ipynb" | \
+	 xargs -P4 -I'{}' $(JUPYTER) nbconvert --execute '{}' \
```

Comment:
is the P4 the multiprocess part? it knows to split the input into 4 sets?

Reply:
P4 is 4 processes in parallel